Dockerfile.lab

```diff
@@ -48,13 +35,11 @@ ENV INCORE=${INCORE} \
 RUN umask 0 && \
   find /opt/conda ! -perm -o=w -print | sort > /tmp/oldfiles && \
   conda config --add channels in-core && \
-  mamba install -c ${CHANNEL} -y \
+  conda install -c ${CHANNEL} -y \
```

Comment:
changed conda in here to mamba and build worked well. Using conda failed to me.

Reply:
any specifics about the error?

Reply:
no it was just the regular build fail when conda was not able to find the package.

Reply:
@robkooper could you also build and let us know if it fails?